sdk/java-sdk-protobuf/src/main/java/kalix/javasdk/workflowentity/WorkflowEntity.java

```diff
@@ -42,6 +45,7 @@
 
 
   private Optional<CommandContext> commandContext = Optional.empty();
+  private Optional<WorkflowEntityContext> workflowEntityContext = Optional.empty();
```

Comment:
Not sure if an additional context is a good way, but extending the current CommandContext won't do the trick, since I also want to use timers in step actions. For the step transition timers are blocked - the context is empty.

Reply:
It seems that the only reason to have `WorkflowEntityContext` is to later be able to access the `MessageCodec` and the `ActorSystem` in other to build the timer. 

So maybe instead we should just have an `Option<TimerScheduler>` that we set when handing a command and running a step transition.

Reply:
I see, but users should use the creation context in that case. That should be the preferred way of getting a handle on the entity id.

Reply:
I mean `WorkflowEntityContext`. Check `KalixSpringApplication` for example. 

Users can let it be injected into constructor when they need to access the `entityId`. Since available in constructor, it will be available in `definition`.

Reply:
Wow, I just realised that the `WorkflowEntityContext` that you were using is the one that we already have. I thought it was called `WorkflowEntityCreationContext`. 

Anyway, `WorkflowEntityContext` is the creation context and is expected to be injected into constructor when we need the entityId. That aligns with what we have done in other entities as well.

Reply:
Ok, now I get it.

Reply:
:) Actually, we've never used it in the constructor, because there is a small bug in KalixSpringApplication, fixing it right now.